devices/gledopto.js

```diff
@@ -487,6 +487,14 @@ module.exports = [
         description: 'Zigbee 10W Floodlight RGB+CCT (pro)',
         extend: gledoptoExtend.light_onoff_brightness_colortemp_color(),
     },
+    {
+        zigbeeModel: ['GL-FL-001P'],
+        model: 'GL-FL-001P',
+        vendor: 'Gledopto',
+        ota: ota.zigbeeOTA,
+        description: 'Zigbee 10W Floodlight RGB+CCT 12V Low Voltage (pro)',
+        extend: gledoptoExtend.light_onoff_brightness_colortemp_color(),
```

Comment:
Please add the color temp range as described on https://www.zigbee2mqtt.io/how_tos/how_to_support_new_devices.html#31-retrieving-color-temperature-range-only-required-for-lights-which-support-color-temperature